src/Business Foundation/App/NoSeries/src/Setup/NoSeriesLine.Table.al

```diff
@@ -131,6 +131,16 @@ table 309 "No. Series Line"
             Caption = 'Temporary Sequence Number';
             DataClassification = SystemMetadata;
             Access = Internal;
+            ObsoleteReason = 'This field is replaced by field 16 - "Temp. Current Sequence No.".';
+            ObsoleteTag = '24.0';
+            ObsoleteState = Removed;
+        }
+        field(16; "Temp. Current Sequence No."; BigInteger)
+        {
+            Caption = 'Temporary Sequence Number';
+            Description = 'The field should only have value during runtime';
```

Comment:
Maybe "This field holds the next sequence number during batching for sequences. It is only used during runtime and no values will be stored in the database.

Reply:
```suggestion
            Description = 'This field holds the next sequence number during batching for sequences. It is only used during runtime and no values will be stored in the database.';
```